src/components/DateTime/DateTime.jsx

```diff
@@ -39,3 +43,58 @@ export function DateTime(props) {
 		</span>
 	);
 }
+
+function validateDateTime(value) {
+	// Return 'Invalid Date' if the input is null or undefined
+	if (value == null) {
+		return 'Invalid Date';
+	}
+
+	// Handle BigInt values explicitly
+	if (typeof value === 'bigint') {
+		// SP-Lang datetime format is represented as BigInt
+		return splDatetimeToIso(value);
+	}
+
+	// Handle string values
+	if (typeof value === 'string') {
```

Comment:
Can be simplified to:

```
if (typeof value === 'string') {
	return new Date(value).toString();
}
```

Reply:
The only thing I removed toString();, because if i don't do it, then further in the code in this place will return Invalid date. Because if i leave toString, then datetime is already a string, not a Date object. So `datetime.toString()` will just call toString() on the string again, not on the Date, and you get Invalid Date, even if the date was valid.

Reply:
makes sense